fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -172,7 +172,7 @@ public void analyze(boolean isOlap) throws AnalysisException {
         }
 
         // disable Bitmap Hll type in keys, values without aggregate function.
-        if (type.isBitmapType() || type.isHllType()) {
+        if (isOlap && type.isBitmapType() || type.isHllType()) {
```

Comment:
Does this affect other types of external tables?

Reply:
I don't think it will affect. In the olap type table, the bitmap and hll fields need to be restricted to only be used as aggregate types, not as keys. All external tables are non-olap type tables, and non-olap type tables do not need to be restricted